mdfreader/mdf.py

```diff
@@ -397,7 +417,7 @@ def setChannelData(self, channelName, data, compression=False):
             temp.compression(data)
```

Comment:
you took under work part of the code, sorry for that.
This has changed and indent is justified